configs/components/yaml-cpp.rb

```diff
@@ -53,6 +53,7 @@
   elsif platform.is_solaris?
     # We always use the i386 build of cmake, even on sparc
     cmake = "#{settings[:basedir]}/i386-pc-solaris2.#{platform.os_version}/bin/cmake"
+    pkg.environment "PATH" => "$$PATH:/opt/csw/bin"
```

Comment:
@kurtwall When I went looking for documentation on this to refer to, I came up short as well. But I can assure you it is necessary to escape the dollar sign by doubling it. A test build without it resulted in:

export PATH="ATH:/opt/csw/bin" && gunzip -c "yaml-cpp-0.5.1.tar.gz"